android/keymanager/src/main/java/io/mosip/registration/keymanager/util/CertificateManagerUtil.java

```diff
@@ -34,6 +36,14 @@ public static boolean isValidCertificateData(String certData) {
         return certData != null && !certData.trim().isEmpty();
     }
 
+    public static boolean isValidApplicationId(String appId) {
+        return appId != null && !appId.trim().isEmpty();
```

Comment:
we should use this https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isBlank-java.lang.CharSequence-m since we already have this library as part of our dependencies